app/src/utils/URLUtils.js

```diff
@@ -13,3 +13,11 @@ export const getRouteFromCurrentPath = (path) => {
   const route = significantPathParts.join("/");
   return route;
 };
+
+export const prefixUrlWithHttps = (url) => {
```

Comment:
Also use this util function in `app/src/views/features/sessions/SessionsIndexPageContainer/SessionsIndexPage/OnboardingView.tsx`